src/material-ui-stories/Progress.stories.tsx

```diff
@@ -261,10 +274,12 @@ export const LinearBuffer = () => {
   );
 };
 
-LinearBuffer.parameters = {
+LinearBufferInternal.parameters = {
```

Comment:
parameters should be applied to the external story itself. Currently, it breaks ignoring this story in chromatic.